client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -14,6 +14,8 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import { recordPageView } from 'state/analytics/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { hasLoadedSites } from 'state/selectors';
```

Comment:
Note that `hasLoadedSites` will return true if `state.sites.items` is not empty. We persist sites, so we can get into the case where we have this information, but the sites fetch for fresh data isn't completed yet. It is still possible for us to get into a state where we have some site information but not for the current siteSlug.

Reply:
Uhm, wondering what could be the case though.
A cached `state.sites.items` should cover all sites owned by a user, and if they then go and create a new site, I assume it will be added to `state.sites.items` and cached consequently.

Reply:
It would be more rare but not too uncommon. For example imagine I use multiple computers, or multiple browser profiles, where in profile A I do not log in for a week or so, and in profile B do a bunch of work adding sites, I log back into profile A, and voila! Isn't storing state fun :D

Reply:
Cache was a mistake!